templates/company/arapp.json

```diff
@@ -4,8 +4,26 @@
       "registry": "0x5f6f7e8cc7346a11ca2def8f827b7a0b612c56a1",
       "appName": "company-template.aragonpm.eth",
       "network": "rpc"
+    },
+    "rinkeby": {
+      "appName": "company-template.aragonpm.eth",
+      "address": "0xe8a5694c9ae3fc73a32e333c3b7b27ccb9a7d2cd",
```

Comment:
Ahh, we don't need to keep the `address` here since it's available from the repo attached to `appName`.

Also, it looks like these addresses were deployed before the new DAOFactory (`0x137676c1941c8b12d18b12eb2215d1361aee20e0`) was deployed; could we redeploy using the `deploy:staging` commands again?